### ISSUE TITLE
Mutable defaults generated from Thrift default values

### ISSUE THREAD

**Issue by bdol** [unmade/thrift-pyi#53]
I have a struct defined in Thrift like the following:

```thrift
struct Datasets {
    1: required set<i64> abc = []
    2: required set<i64> xyz = []
}
```

This is compiled to something like:
```python
@dataclass
class Datasets:
    abc: Set[int] = set()
    xyz: Set[int] = set()
```

However, I get a error when trying to instantiate an instance of this class:
```
# ValueError: mutable default <class 'set'> for field a is not allowed: use default_factory
```

I think the simple fix for this is to compile to using `default_factory`, similar to [this](https://stackoverflow.com/a/75677129) SO answer. Thanks!

**Comment by unmade** [comment]
Hey,

Thanks for a good case. I fixes it, please check out new `1.1.0` version. Feel free to re-open the issue if something wrong

**Comment by bdol** [comment]
Awesome, thank you!
